service/qos/raft_service_level_distributed_data_accessor.cc

```diff
@@ -10,10 +10,15 @@
 #include "cql3/query_processor.hh"
 #include "db/consistency_level_type.hh"
 #include <seastar/core/abort_source.hh>
+#include "seastar/util/variant_utils.hh"
+#include "service/raft/group0_state_machine.hh"
```

Comment:
>  service/qos/raft_sl_dda: use service_levels_change when enabled
> 
> When RAFT_SERVICE_LEVELS_CHANGE is enabled, use new `service_levels_change` change.
> Otherwise, use old `write_mutations`.

Nit: this description lacks explanations. Why do we want to use the new `service_levels_change`?
(I know why, it's just that it should be explained in the description)